tests/unit/handler.unitTest.ts

```diff
@@ -164,6 +164,66 @@ describe("The lambda function handler", () => {
         expect(updateEuVehicleMock.mock.calls[0][1]).toEqual("m1");
       });
 
+      it("should call the /vehicles/add-provisional function with correct event payload", async () => {
+        const payload = {
```

Comment:
could be moved into a beforeEach I think?

Reply:
moved to the start of the file as the payload is not mutated through the suite.

Reply:
would affect the whole unit test as same payload is used. I would suggest as a tech debt activity.

Reply:
agreed, that is something we can note for now as this was part of the codebase and possibly look at having the set up in a beforeEach later.